### PR TITLE
docs(aio): fix typos in resources.json

### PR DESCRIPTION
**Pull request by sibiraj-s**
This is a copy of PR #17481. Since the branch is deleted for that PR. I was unable to make changes to that PR as requested. So I created this.

### CODE REVIEW

**Comment by Sibiraj-S** [comment]
+1

File: aio/content/marketing/resources.json [hunk 1]
```diff
@@ -568,7 +568,7 @@
             "url": "http://ninja-squad.com/formations/formation-angular2"
           },
           "a2b": {
-            "desc": "Angular Boot Camp covers introductory and intermediate content. It includes extensive workshop session, with hands-on help from our experienced developer-trainers. At the end of this class, student are usually able to use AngularJS to make an end-to-end, working application.",
+            "desc": "Angular Boot Camp covers the introductory and intermediate content. It includes an extensive workshop session, with hands-on help from our experienced developer-trainers. At the end of this class, students are usually able to use AngularJS to make an end-to-end, working application.",
```
Comment (wardbell) [inline aio/content/marketing/resources.json]: Should not add the "the" in the first sentence; the original was correct as it describes a quality of the content, not specific content. Please revert.

Your addition of "an" is correct.

File: aio/content/marketing/resources.json [hunk 2]
```diff
@@ -395,7 +395,7 @@
             "url": "https://leanpub.com/router"
           },
           "vsavkin-essential-angular": {
-            "desc": "The book is a short, but at the same time, fairly complete overview of the key aspects of Angular written by its core contributors Victor Savkin and Jeff Cross. The book will give you a strong foundation. It will help you put all the concepts into right places. So you will get a good understanding of why the framework is the way it is.",
+            "desc": "The book is short, but at the same time, fairly complete overview of the key aspects of Angular written by its core contributors Victor Savkin and Jeff Cross. The book will give you a strong foundation. It will help you put all the concepts into right places. So you will get a good understanding of why the framework is the way it is.",
```
Comment (wardbell) [inline aio/content/marketing/resources.json]: If we're going to fix this, let's fix it properly
>This book by core contributors, Victor Savkin and Jeff Cross, is a short overview of the key aspects of Angular. You'll acquire a strong foundation in Angular concepts and gain insight into why the framework is the way it is.",

**Review by wardbell — APPROVED** [review]
Thanks. Almost there (and I'm LGTM-ing anyway.  Please make the small changes I suggested and I'll get it merged.

**Comment by Sibiraj-S** [comment]
@wardbell . updated

**Comment by Sibiraj-S** [comment]
can merge and close the PR?

**Pull request closed without merging**